pkg/kube/wrappers/ingress-wrapper.go

```diff
@@ -59,21 +60,26 @@ func (iw *IngressWrapper) getIngressSubPathWithPort() string {
 	return port + subPath
 }
 
-func (iw *IngressWrapper) getIngressPort() string {
-	rule := iw.Ingress.Spec.Rules[0]
-	if rule.HTTP != nil {
-		if rule.HTTP.Paths != nil && len(rule.HTTP.Paths) > 0 {
-			return rule.HTTP.Paths[0].Backend.ServicePort.StrVal
-		}
-	}
-	return ""
-}
+// func (iw *IngressWrapper) getIngressPort() string {
```

Comment:
Because it was trying to build the wrong monitor url if there was a named ServicePort in ingress. And anyways the service port is not needed in monitorUrl